insights/parsers/ls_usr_sbin.py

```diff
@@ -0,0 +1,46 @@
+"""
+LsUsrSbin - command ``ls -ln /usr/sbin``
+========================================
+
+The ``ls -ln /usr/sbin`` command provides information for the listing of the
+``/usr/sbin`` directory.
+
+Sample input is shown in the Examples. See ``FileListing`` class for
+additional information.
+
+Sample directory list::
+
+    total 41472
+    -rwxr-xr-x. 1 0  0   11720 Mar 18  2014 accessdb
+    -rwxr-xr-x. 1 0  0    3126 Oct  4  2013 addgnupghome
+    -rwxr-xr-x. 1 0  0   20112 Jun  1  2017 addpart
+    -rwxr-xr-x. 1 0  0  371912 Jan 27  2014 postconf
+    -rwxr-sr-x. 1 0 90  218552 Jan 27  2014 postdrop
+
+Examples:
+
+    >>> add_filter(Specs.ls_usr_sbin, "accessdb")
+    >>> "accessdb" in ls_usr_sbin
+    False
+    >>> "/usr/sbin" in ls_usr_sbin
+    True
+    >>> ls_usr_sbin.dir_entry('/usr/sbin', 'accessdb')['type']
+    '-'
+
+"""
+
+
+from insights.core.filters import add_filter
+from insights.specs import Specs
+
+from .. import CommandParser, parser
+from .. import FileListing
+
+
+add_filter(Specs.ls_usr_sbin, "total")
```

Comment:
@TripleJQK if this is being filtered for just this information then shouldn't it just be a `CommandParser` and not a `FileListing` parser?  The filter would probably cause the `FileListing` parser to fail and no data would be provided.

Reply:
@bfahr hi, thank you for your advice.
At first, I put two filter here:
~~~
add_filter(Specs.ls_usr_sbin, "total")
add_filter(Specs.ls_usr_sbin, "postdrop")
~~~
for more, please see: https://github.com/RedHatInsights/insights-core/pull/1257/commits/b6fce0a63bf2f56c68660851c3bad9cce78e07ea

But by xiangce's advice, he thought:"I think it's OK to add the 'total' here, but better add the 'postdrop' in the corresponding plugins, since different plugins require a different filter." So I delete the postdrop filter from the parser and will add it to the corresponding plugin(which is not coded yet, as pr is not merged). @bfahr do think it is good to do it as xiangce's way? I think other other parser like ps are also in this way

Reply:
@bfahr I have make a plugin code with only filter first here:
https://github.com/RedHatInsights/insights-plugins/pull/959